### PR TITLE
citra_qt: Migrate logging macros

### PR DESCRIPTION
**Pull request by daniellimws**
Follow-up of #3533

Replace prefix of all logging macros from LOG to NGLOG

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3574)

### CODE REVIEW

File: src/citra_qt/main.cpp [hunk 1]
```diff
@@ -768,8 +766,7 @@ void GMainWindow::OnGameListOpenFolder(u64 program_id, GameListOpenTarget target
         return;
     }
 
-    LOG_INFO(Frontend, "Opening %s path for program_id=%016" PRIx64, open_target.c_str(),
-             program_id);
+    NGLOG_INFO(Frontend, "Opening {} path for program_id={:#016}", open_target, program_id);
```
Comment (MerryMage) [inline src/citra_qt/main.cpp]: drop the `#` (we don't usually prefix program ids with 0x)

File: src/citra_qt/main.cpp [hunk 2]
```diff
@@ -768,8 +766,7 @@ void GMainWindow::OnGameListOpenFolder(u64 program_id, GameListOpenTarget target
         return;
     }
 
-    LOG_INFO(Frontend, "Opening %s path for program_id=%016" PRIx64, open_target.c_str(),
-             program_id);
+    NGLOG_INFO(Frontend, "Opening {} path for program_id={:016}", open_target, program_id);
```
Comment (wwylele) [inline src/citra_qt/main.cpp]: Should be `{:016X}`?

Comment (daniellimws) [reply src/citra_qt/main.cpp]: I'm not sure. Are we printing the program id as hex values?

Comment (wwylele) [reply src/citra_qt/main.cpp]: Yes, as PRI*x*64 is for hex values

Comment (daniellimws) [reply src/citra_qt/main.cpp]: Oh I somehow saw `PRId64` :confused:

Ok.

**Pull request merged**